include/boost/beast/http/error.hpp

```diff
@@ -153,7 +153,16 @@ enum class error
         a new parser for each message. This can be easily done by
         storing the parser in an boost or std::optional container.
     */
-    stale_parser
+    stale_parser,
+
+    /** The body is shorter than claimed
```

Comment:
Missing punctuation, and "claimed" is borderline non-sterile. "Claiming" is usually done by people. I would say "The message body is shorter than expected."